docs/components/novabotflow.md

```diff
@@ -8,6 +8,48 @@
 #Nova.BotFlow
 ##Product Notices
 ***
+
+###2022R4-5 :briefcase: Epic
+####New Features
+- *Nova.Botflow*: Add "Video" processing (FB Messenger) [:clipboard: NC2-177] [NC2-177]
+
+	- Added processing of outgoing videos to the code
+	> .\src\node-red-contrib-chatbot\lib\platforms\facebook\facebook.js
+
+- *Nova.Botflow*: IVR menu: chatBot changes [:clipboard: NC2-368] [NC2-368]
+	- Updated botflow according to new [Menu (IVR) specification](https://drive.google.com/open?id=1wl5w_mNff59Cdrz0BJfL9iI3zrFnDCjw&authuser=cristina.podoliuh%40novait.com.ua&usp=drive_fs).
+
+- *Nova.Botflow*: NodeRed 3 migration [:clipboard: NC2-404] [NC2-404]
+	- Updated **node-red -> 3.0.2**
+	- Updated **node.js -> 18.12.1** (node:18.12.1-alpine)
+	- Updated **yarn.lock**
+	- Added [commitlint](https://commitlint.js.org/).
+
+
+####Bug Fixes
+
+- *Nova.Botflow*: Correct handling of unsupported types in connectors[:clipboard: NC2-364] [NC2-364]
+	- Fixed handling of messages with type = undefined if the message passed all checks and did not receive a type, so this type is not supported.
+	- Such messages are reset and do not leave the connector.
+
+- *Nova.Botflow*: Queue modernization [:clipboard: NC2-398] [NC2-398]
+
+	- Fixed incorrect behavior when the queue gap is set to 0
+	- Now we can insert a Queue Selector into the several nesting levels. For example: payload.chatId.
+
+	
+- *Nova.Botflow*: Errors with sending pictures, videos and messages [:clipboard: NC2-293] [NC2-293]
+	- In TG\Viber\Messenger\Nexmo\Whatsapp Buisness nodes added variable **attachmentUrl** in **payload**.
+	- When the file size is greater than **MAX_FILE_SIZE**(env) Chatwoot In (NovaTalks In) sends **attachmentUrl**.
+	
+- *Nova.Botflow*: NodeRed fallDown while sending **msg** without **chat** [:clipboard: NC2-416] [NC2-416]
+	- Added processing **msg** objects without **chat()** by **done()** func.
```

Comment:
- This is not a clear and precise description
- Recheck your grammar, you forgot articles